File: back/src/main/java/mitei/mitei/political/balancesheet/manage/kanrensha/dto/riyousha/SaveRiyoushaManagerCapsuleDto.java
package mitei.mitei.political.balancesheet.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import mitei.mitei.political.balancesheet.manage.kanrensha.dto.FrameworkCapsuleDto;

/**
 * 運営者ユーザー格納Dto
 */
public class SaveRiyoushaManagerCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 利用者運営者Dto */
    private RiyoushaManagerDto riyoushaManagerDto = new RiyoushaManagerDto();

    /**
     * 利用者運営者Dtoを設定する
     *
     * @return 利用者運営者Dto
     */
    public RiyoushaManagerDto getRiyoushaManagerDto() {
        return riyoushaManagerDto;
    }

    /**
     * 利用者運営者Dtoを取得する
     *
     * @param riyoushaManagerDto 利用者運営者Dto
     */
    public void setRiyoushaManagerDto(final RiyoushaManagerDto riyoushaManagerDto) {
        this.riyoushaManagerDto = riyoushaManagerDto;
    }

}

File: front/src/dto/riyousha/saveRiyoushaManagerCapsuleDto.ts
import FrameworkCapsuleDto from '../frameworkCapsuleDto';
import RiyoushaManagerDto from './riyoushaManagerDto';

export default interface SaveRiyoushaManagerCapsuleInterface {
}

/**
 * 運営者ユーザー格納Dto
 */
export default class SaveRiyoushaManagerCapsuleDto extends FrameworkCapsuleDto
  implements SaveRiyoushaManagerCapsuleInterface {

  /** 利用者運営者Dto */
  riyoushaManagerDto: RiyoushaManagerDto;

  constructor() {
    super();
    this.riyoushaManagerDto = new RiyoushaManagerDto;
  }
}